cloudprovision.json

```diff
@@ -0,0 +1,139 @@
+{
+  "create": {
+    "steps": [
+      {
+        "id": "clone solution repo",
+        "name": "gcr.io/cloud-builders/git",
+        "entrypoint": "bash",
+        "args": [
+          "-c",
+          "git clone -b feature/enable-cloud-provision https://github.com/gitrey/quota-monitoring-solution.git"
+        ]
+      },
+      {
+        "id": "replace tf vars",
+        "name": "ubuntu",
+        "entrypoint": "bash",
+        "env": [
+          "PROJECT_ID=$PROJECT_ID",
+          "_APP_ID=$_APP_ID",
+          "_APP_NAME=$_APP_NAME",
+          "_REGION=$_REGION",
+          "_SERVICE_ACCOUNT=$_SERVICE_ACCOUNT",
+          "_BUCKET_NAME=$_BUCKET_NAME",
+          "_NOTIFICATION_EMAIL=$_NOTIFICATION_EMAIL",
+          "_THRESHOLD=$_THRESHOLD"
+        ],
+        "args": [
+          "-c",
+          "sed -i s/PROJECT_ID/$PROJECT_ID/g quota-monitoring-solution/terraform/example/terraform.tfvars"
```

Comment:
This will need to be done for all env vars in here https://github.com/gitrey/quota-monitoring-solution/pull/1/files#diff-5deabc47686738f43ae067bbfc4ab6389d42396868e1f5a7ea102076d121c777R18-R25